### PR TITLE
Updated the port numbers and the paths in the scripts to match the ones used in client itests

### PR DESCRIPTION
**Pull request by sanjay-saxena**
This would make it easier to run the client itests against the scripts, eventually.

### CODE REVIEW

**Comment by cmebarrow** [comment]
@sanjay-saxena Did you run this by @jfallows ?

**Comment by sanjay-saxena** [comment]
This will not be merged as-is because @jfallows likes to move the empty frames for backward compatibility out of the spec. Once we have a consensus, we will make the changes accordingly. Look at #299 for the on-going conversation.

**Comment by sanjay-saxena** [comment]
@jfallows - Please hold on the merge as this is not as simple as we originally thought. I will update the [ticket](https://github.com/kaazing/roadmap/issues/669) with details. Based on my findings, we will need more changes to the specification scripts and SPEC.md.

File: specification/httpx/src/main/scripts/org/kaazing/specification/httpx/extended/client.sends.message.between.opening.and.extended.handshake/request.rpt [hunk 1]
```diff
@@ -15,7 +15,7 @@
 #
 
 property writeMask ${httpx:randomBytes(4)}
```
Comment (dpwspoon) [inline specification/httpx/src/main/scripts/org/kaazing/specification/httpx/extended/client.sends.message.between.opening.and.extended.handshake/request.rpt]: I think this was a mistake.  We should use standard same ports and paths ("8080" and "path"), just like all the other specification tests.  @sanjay-saxena can you back out this change, and switch to 8080 and path

**Pull request merged**